apps/atomy-q/API/database/factories/ComparisonRunFactory.php

```
@@ -0,0 +1,84 @@
+
+    public function preview(): static
+    {
+        return $this->state(fn (array $attributes) => [
+            'status' => 'draft',
+            'is_preview' => true,
+        ]);
+    }
+
+    public function forRfq(Rfq $rfq): static
+    {
+        return $this->state(fn (array $attributes) => [
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$attributes'. (undefined)

(UnusedFormalParameter)